### ISSUE TITLE
create/update time duplicated in doc JSON

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#2939]
```json
{
    "type": "Pipeline",
    "uuid": "8c9b1d43-ccb0-4050-89d9-3aa0980d60ed",
    "name": "Example Solr index",
    "version": "3ca2ce8b-7c7c-4900-8a16-34444666fae9",
    "createTime": 1656000709218,
    "updateTime": 1656000709218,
    "createUser": "admin",
    "updateUser": "admin",
    "parentPipeline": {
        "type": "Pipeline",
        "uuid": "fcef1b20-083e-436c-ab95-47a6ce453435",
        "name": "Indexing"
    },
    "updateTimeMs": 1656000709218,
    "createTimeMs": 1656000709218
}
```

Likely caused by the name of the getters on `Doc` being named like `getCreateTimeMs` which is different from the field `createTime`.

Once fixed we may need to clean the data by removing the dup json keys.

**Comment by at055612** [comment]
SQL to check for any mismatched values
```sql
select
    *
from
(select
	v.id,
    v.uuid,
    v.name,
    v.content_extension,
    json_extract(v.meta_data, '$.type') doc_type,
    from_unixtime(json_extract(v.meta_data, '$.createTime')/1000) createTime,
    from_unixtime(json_extract(v.meta_data, '$.createTime')/1000) createTimeMs,
    from_unixtime(json_extract(v.meta_data, '$.updateTime')/1000) updateTime,
	from_unixtime(json_extract(v.meta_data, '$.updateTime')/1000) updateTimeMs
from 
	(select
		dm.id,
		dm.uuid,
		dm.name,
		dd.ext content_extension,
		convert(dm.data using utf8) meta_data,
		convert(dd.data using utf8) content_data
	from doc dm
	inner join doc dd on dm.uuid = dd.uuid and dd.ext != dm.ext
	where dm.ext = 'meta') v) v2
where v2.createTime != v2.createTimeMs
OR v2.updateTime != v2.updateTimeMs;
```

**Comment by at055612** [comment]
Fixed in >7.0-beta.215
